segmentstore/storage/impl/src/test/java/io/pravega/segmentstore/storage/impl/extendeds3/S3FileSystemImpl.java

```diff
@@ -154,12 +155,20 @@ public CopyPartResult copyPart(CopyPartRequest request) {
 
     @Override
     public void deleteObject(String bucketName, String key) {
-
+        Path path = Paths.get(this.baseDir, bucketName, key);
+        try {
+                Files.delete(path);
+        } catch (IOException e) {
+            throw new S3Exception("NoSuchKey", HttpStatus.SC_NOT_FOUND, "NoSuchKey", "");
+        }
     }
 
     @Override
     public DeleteObjectsResult deleteObject(DeleteObjectsRequest request) {
```

Comment:
The request is to delete multiple objects, shouldn't the call be called `deleteObjects` to differentiate from the one that deletes a single object? I also checked the `S3ImplBase` interface, and that doesn't seem to be following any imported interface. Shouldn't we be implementing the extended S3 interface?